doc/source/regression/header.md

```diff
@@ -0,0 +1,15 @@
+````{tip}
+:class: pywt-margin-top-0
+
+This page can also be run or downloaded as a Jupyter notebook.
+
+- Run in a new tab using JupyterLite[^what-is-lite]:
+  ```{jupyterlite} {{ parent_docname }}.ipynb
```

Comment:
```suggestion
  ```{notebooklite} {{ parent_docname }}.ipynb
```

With `jupyterlite-sphinx` 0.17.1, we can now use the Notebook interface, which is much nicer for displaying an individual notebook.

Reply:
Oh, nice! That was released two weeks ago, about when I started working on this. I remember trying the `notebooklite` directive first but it wasn't working locally, probably because I had an older version of jupyterlite-sphinx installed.

Reply:
I'll update all of the directives